op-chain-ops/upgrades/check.go

```diff
@@ -123,16 +123,17 @@ func cmpVersion(v1, v2 string) bool {
 	return v1 == v2
 }
 
-// ToSuperchainName turns a base layer chain id into a superchain network name.
-func ToSuperchainName(chainID uint64) (string, error) {
-	if chainID == 1 {
-		return "mainnet", nil
-	}
-	if chainID == 5 {
-		return "goerli", nil
-	}
-	if chainID == 11155111 {
-		return "sepolia", nil
-	}
-	return "", fmt.Errorf("unsupported chain ID %d", chainID)
+// SuperChainID returns the chain ID for the supplied superchain name, or an error
+// if no such superchain is known.
+func SuperChainID(superchain string) (uint64, error) {
```

Comment:
This function is a bit of duct tape, this functionality should ideally live in the registry, like if a new target is added then we may forget to update this and there are no checks on this function in CI

Reply:
Agreed. I removed this in [b3e4bbf](https://github.com/ethereum-optimism/optimism/pull/9203/commits/b3e4bbfd60bd3267542f4cc9cea2447cb36db621).